bchec/signature.go

```diff
@@ -448,9 +522,37 @@ func signRFC6979(privateKey *PrivateKey, hash []byte) (*Signature, error) {
 	return &Signature{R: r, S: s}, nil
 }
 
+// signSchnorr signs the hash using the schnorr signature algorithm.
+func signSchnorr(privateKey *PrivateKey, hash []byte) (*Signature, error) {
+	additionalData := []byte{'S', 'c', 'h', 'n', 'o', 'r', 'r', '+', 'S', 'H', 'A', '2', '5', '6', ' ', ' '}
```

Comment:
Maybe a comment about this requiring 16 bytes? Or why you chose this value?

Reply:
for reference : https://github.com/Bitcoin-ABC/bitcoin-abc/blob/57334abef9fb6564b21a00be60cf844e81018f7c/src/secp256k1/src/modules/schnorr/schnorr_impl.h#L182
also: https://github.com/bitcoincashorg/bitcoincash.org/blob/master/spec/2019-05-15-schnorr.md#recommended-practices-for-secure-signature-generation